_sass/components/_list.scss

```diff
@@ -1,3 +1,14 @@
+ul {
+  li {
+    line-height: 2.25rem;
+    ul {
+      list-style-type: none;
+      list-style-image: url("../img/list-bullet-nested.svg");
```

Comment:
From what I can tell, it's unfortunately not possible to vertically align a `list-style-image`, which has the unfortunate side-effect of not being centered with the text, instead appearing toward the top of the line. One option could be to include the padding in the image itself, so that the bullet image is centered in an image of 2.25rem height.